### ISSUE TITLE
Even though we restrict the files to specific users on the Topcoder Team, they can access the files

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#3889]
### Expected behavior
Should apply the restriction to that specific user

### Actual behavior
Even though we restrict the files to specific users on the Topcoder Team, they can access the files

### Steps to reproduce the problem

- Login as Manager
- Select a project from the list {https://connect.topcoder-dev.com/projects/9074}
- Go to 'Dashboard' > Expand a phase (QA Iteration)
- Go to 'Specification' tab
- Upload a file {CSVFile_119kb (1).csv} with "Restrict access to" > TCConnCopilot
- Login as Copilot
- Select a project from the list {https://connect.topcoder-dev.com/projects/9074}
- Go to 'Dashboard' > Expand a phase (QA Iteration)
- Go to 'Specification' tab
- Download the CSVFile_119kb (1).csv file

### Screenshot/screencast

https://monosnap.com/file/gY8UhWzKZWHpz4QOvuV6m3kqd4B8MI

#### Environment
**- OS:** HP 15-dw1xxx 15.6 inch; Window 10
**- Browser (w/version):** FF 75 | Chrome 80.0.3987.163
**- User role (client, copilot or manager):** Copilot
**- Account used:**  `topcoderconnect+Copilot@gmail.com`

**Comment by vikasrohit** [comment]
@codejamtc I didn't understand the issue completely. It seems the files were shared with `TCConnCopilot` and you used that same account to download the file which means it is working fine. Let me know if I misunderstood the issue.

**Comment by codejamtc** [comment]
@vikasrohit 0:21 The Manager restrict the CSV file for `TCConnCopilot` and 0:29 when we logged in as `TCConnCopilot` user can see that CSV file

**Comment by vikasrohit** [comment]
I think you get it wrong. `Restrict Access To` means only the listed users are allowed to access the file, no one else. And if we don't have any thing listed there, it would be shared with all project members.

**Comment by codejamtc** [comment]
@vikasrohit Thanks for the clarification. Looking at your comment https://github.com/appirio-tech/connect-app/issues/3889#issuecomment-613217775 I think this is working properly, But I will re-check and close if it is working properly

**Comment by codejamtc** [comment]
@vikasrohit There is something wrong here.

https://connect.topcoder-dev.com/projects/9074

I have logged in as `Manager` and for the phase `QA Iteration F` I have added few files. After that I gave;

CSVFile_119kb (1).csv = RESTRICT ACCESS TO `TCConAccountMgr`

Then I have logged in as `TCConAccountMgr` and check the `QA Iteration F` phase. I can't see the Specification tab

https://monosnap.com/file/CjTPWceV0vzeb4KVIBFl6OX5N1lBy2

**Comment by maxceem** [comment]
@codejamtc that's correct behavior, not all users can see the `Specification` tab of the phases.

**Comment by codejamtc** [comment]
@maxceem Noted, Thanks
